### PR TITLE
Add taplo to support toml formatting

### PR DESCRIPTION
**Pull request by julian-bitkey**

### CODE REVIEW

**Comment by julian-bitkey** [comment]
@quad 

Tested this locally on my Mac, if there is an easy way to do it on a Linux box I'm happy to do it before it goes in.

**Comment by alecthomas** [comment]
CI will test it on Linux.

File: taplo.hcl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"
+    on unpack {
+      rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
+    }
+}
+
+linux {
+  source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"
+  on unpack {
+    rename { from = "${root}/taplo-${os}-{$xarch}" to = "${root}/taplo" }
```
Comment (quad) [inline taplo.hcl:16]:
```suggestion
    rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
```

**Review by quad — APPROVED** [review]

File: taplo.hcl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
+    }
+}
+
+linux {
+  source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"
+  on unpack {
+    rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
+  }
+}
```
Comment (quad) [inline taplo.hcl:18]: Wait a second. Aren't these identical???
```suggestion
source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"

on unpack {
  rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
}
```

**Review by quad — APPROVED** [review]
LGTM!

**Pull request merged**